app/frontend/resources/views/items/show.blade.php

```diff
@@ -178,6 +178,14 @@ class="mw-100">
 </div>
 @endsection
 
+@section('script')
+<script>
+    var SimpleLightbox = window.SimpleLightbox;
+    new SimpleLightbox({elements: '.item-image-columns a'});
```

Comment:
I wonder if we should add data-tags to images explicitly, e.g. `data-lightbox="show"` and `[data-lightbox="show"]` as the query?

Reply:
depending on if we want to do something with it too we may wanna also store a reference to the lightbox on the page? (honestly this one doesn't matter too much)

Reply:
data tags might be worthwhile, yeah - we could move this to bootstrap.js in that case instead of as a separate include on the item pages (I wasn't sure what else used the `item-image-columns` class and didn't want rogue lightboxes)